subprojects/gdk_rust/tests/integration.rs

```diff
@@ -123,9 +123,25 @@ fn liquid() {
     test_session.stop();
 }
 
+fn check_account_balances(test_session: &TestSession, balances: &HashMap<u32, u64>) {
+    for (n, balance) in balances {
+        println!("{}: {}", n, balance);
```

Comment:
Was this meant to stay here? Maybe use the `log` macros instead of `println`?

Reply:
Ops, it was not. Removed